qsimcirq_tests/qsimcirq_test.py

```diff
@@ -67,7 +67,7 @@ def test_repeated_keys():
         cirq.measure(q, key="m"),
     )
     result = qsimcirq.QSimSimulator().run(circuit, repetitions=10)
-    assert result.records["m"] == (10, 3, 1)
+    assert result.records["m"].shape == (10, 3, 1)
```

Comment:
Could you add tests for reusing measurement keys with:
- different qubits, different moment (ok)
- different qubits, same moment (error)
- different # of qubits (error)
- CircuitOperation(use_repetition_ids=False) (ok)